integration_tests/backend/test_reads.py

```diff
@@ -680,6 +683,28 @@ def test_endpoint_node_metric_result_content_get(self):
             # One of these should be successful (direct descendent of operator)
             assert not resp_obj.is_downsampled
             assert len(resp_obj.content) > 0
+            
+    def test_endpoint_node_metric_results_get(self):
+        flow_id = self.flows["flow_with_metrics_and_checks"][0]
+        flow = self.client.flow(flow_id)
+        workflow_resp = flow._get_workflow_resp()
+        dag_id = workflow_resp.workflow_dag_results[0].workflow_dag_id
+        dag_result_id = workflow_resp.workflow_dag_results[0].id
+
+        dag_result_resp = globals.__GLOBAL_API_CLIENT__.get_workflow_dag_result(
+            flow_id,
+            dag_result_id,
+        )
+        metric_artifact_id = None
+        for artifact_id, artifact in dag_result_resp.artifacts.items():
```

Comment:
Is there an API to directly get by the artifact name?  I feel it's a little obscure to reader that these lines are trying to find the metric artifact

Reply:
Unfortunately, I don't think so. I've added a comment explaining what the lines do to make it less obscure.